README.md

```diff
@@ -1,13 +1,14 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_tree-from-object-DOM/)
+   - [DEMO LINK](https://MaxF1996.github.io/js_tree-from-object-DOM/)
```

Comment:
Add the demo link to the PR description 
<img width="949" alt="image" src="https://github.com/user-attachments/assets/de97422a-a479-4ab4-bbd7-8cf78e0d81b3">